src/constants.ts

```diff
@@ -1,5 +1,55 @@
 import * as _ from 'lodash';
 
+_.set(window, 'OPENSHIFT_CONSTANTS.HELP_BASE_URL', "https://docs.openshift.org/latest/");
+var HELP: any = {
```

Comment:
I'm concerned about copying all of these into this repository. How are we going to keep this in sync across repos?